tools/src/h5perf/CMakeLists.txt

```diff
@@ -23,41 +22,14 @@ set_global_variable (HDF5_UTILS_TO_EXPORT "${HDF5_UTILS_TO_EXPORT};h5perf_serial
 
 set (H5_DEP_EXECUTABLES h5perf_serial)
 
-#-----------------------------------------------------------------------------
-# Add Target to clang-format
-#-----------------------------------------------------------------------------
 if (HDF5_ENABLE_FORMATTERS)
   clang_format (HDF5_TOOLS_SRC_H5PERF_h5perf_serial_FORMAT h5perf_serial)
 endif ()
 
+# --------------------------------------------------------------------
+# h5perf
+# --------------------------------------------------------------------
 if (H5_HAVE_PARALLEL)
-  if (UNIX)
-    #-- Adding test for perf - only on unix systems
-    set (perf_SOURCES
-        ${HDF5_TOOLS_SRC_H5PERF_SOURCE_DIR}/perf.c
-    )
-    add_executable (perf ${perf_SOURCES})
-    target_include_directories (perf PRIVATE "${HDF5_TEST_SRC_DIR};${HDF5_SRC_DIR};${HDF5_SRC_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
-    if (NOT ONLY_SHARED_LIBS)
-      TARGET_C_PROPERTIES (perf STATIC)
-      target_link_libraries (perf PRIVATE ${HDF5_TOOLS_LIB_TARGET} ${HDF5_LIB_TARGET} "$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_LIBRARIES}>")
-    else ()
-      TARGET_C_PROPERTIES (perf SHARED)
-      target_link_libraries (perf PRIVATE ${HDF5_TOOLS_LIBSH_TARGET} ${HDF5_LIBSH_TARGET} "$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_LIBRARIES}>")
-    endif ()
-    set_target_properties (perf PROPERTIES FOLDER perform)
-    set_global_variable (HDF5_UTILS_TO_EXPORT "${HDF5_UTILS_TO_EXPORT};perf")
-
-    set (H5_DEP_EXECUTABLES perf)
-
-    #-----------------------------------------------------------------------------
-    # Add Target to clang-format
-    #-----------------------------------------------------------------------------
-    if (HDF5_ENABLE_FORMATTERS)
-      clang_format (HDF5_TOOLS_SRC_H5PERF_perf_FORMAT perf)
-    endif ()
-  endif ()
-
   #-- Adding test for h5perf
```

Comment:
Might as well remove this since we are building programs - not adding tests.

Reply:
Fixed